ee/clickhouse/queries/funnels/funnel_event_query.py

```diff
@@ -16,7 +17,7 @@ def get_query(self, entities=None, entity_name="events", skip_entity_filter=Fals
                 if self._column_optimizer.should_query_elements_chain_column
                 else ""
             ),
-            f"{self.DISTINCT_ID_TABLE_ALIAS}.person_id as person_id" if self._should_join_distinct_ids else "",
+            f"{get_aggregation_target_field(self._filter.aggregation_group_type_index, self.EVENT_TABLE_ALIAS, self.DISTINCT_ID_TABLE_ALIAS)} as person_id",
```

Comment:
As mentioned, this would be nice to rename in a follow-up :)

Reply:
Will come in soon after this goes in!: https://github.com/PostHog/posthog/pull/6939